### PR TITLE
Add Finnish Language

### PR DESCRIPTION
**Pull request by JeshikaDev**
i added finnish translation it should be correct but if i have some parts wrong feel free to fix them.

### CODE REVIEW

**Comment by micwoj92** [comment]
>but if i have some parts wrong feel free to fix them.

I think that most help you would get will be suggestions from google translate ;p
I'll try to review tomorrow morning.

**Comment by JeshikaDev** [comment]
> I think that most help you would get will be suggestions from google translate ;p

if you want more accurate translations you should use https://www.deepl.com/ just a little tip but its my personal opinion

**Comment by micwoj92** [comment]
I am aware of that website, what I meant is that most of us don't know Finnish :P

**Comment by JeshikaDev** [comment]
i see makes sense.

File: Langs/040B.lang [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+Boot Method=Käynnistys Tapa
+CPU Compatibility=Processorin Yhteensopivuus
+CPU Core Count=Processorin Ytimien Määrä
+CPU Frequency=Processori Kellotaajuus
+DirectX + WDDM2=DirectX + WDDM2
+Disk Partition Type=Levyn Osio Tyyppi
+RAM Installed=Ramia Asennettuna
+Secure Boot=Turvallinen Käynnistys
+Storage Available=Tilaa Vapaana
+TPM Version=TMP versio
```
Comment (micwoj92) [inline Langs/040B.lang:22]: typo?

File: Langs/040B.lang [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+Checking...=Varmistetaan...
+64 Bit CPU=x64 Bittinen Processori
+64 Bit OS=x64 Bittinen Käyttöjärjestelmä
```
Comment (micwoj92) [inline Langs/040B.lang:25]: should rather be 64-bit

File: Langs/040B.lang [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+32 Bit CPU=x32 Bittinen Processori
+32 Bit OS=x32 Bittinen Käyttöjärjestelmä
```
Comment (micwoj92) [inline Langs/040B.lang:27]: same here, for consistency

File: Langs/040B.lang [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+Unable to Determine=Ei Voida Määrittää
+Unable to Check List=Ei Voida Katsoa Listaa
+Error Accessing List=Ongelma Avatessa Listaa
+Not Currently Listed as Compatible=Ei Tällä Hetkellä Tuoettu
+Listed as Compatible=Listattu Tuetuksi
+Cores=Ytimiä
+Threads=Lanjoka
```
Comment (micwoj92) [inline Langs/040B.lang:34]: This was untranslatable on deepl and on google, both suggested "kierteet" but I think it means threads like on nuts and bolts.

**Review by micwoj92 — CHANGES_REQUESTED** [review]

File: Langs/040B.lang (hunk 4 above)
Comment (JeshikaDev) [reply Langs/040B.lang:34]: well raw translation of Thread is "Lanka" then Threads would be Lankoja that was a mistype indeed.

File: Langs/040B.lang (hunk 2 above)
Comment (JeshikaDev) [reply Langs/040B.lang:25]: well i guess i can do that but actual raw translation for 64-bit is 64 bittinen

File: Langs/040B.lang (hunk 1 above)
Comment (JeshikaDev) [reply Langs/040B.lang:22]: well no there is no typos the language itself is just weird

Comment (JeshikaDev) [reply Langs/040B.lang:22]: yeah there is typo

**Review by JeshikaDev — COMMENTED** [review]
i added couple of typo fixes and changed that thing about the "Bittinen"

**Comment by micwoj92** [comment]
Thank you for fixing!

File: Langs/040B.lang [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+GPT Not Detected=GPT ei havaittu
+Enabled=Päällä
+Supported=Tuoettu
+Not Supported=Ei Tuoettu
+Disabled / Not Detected=Ei Päällä / Tuoettu
+Drive(s) Meet Requirements=Levyä Täyttää Vaatimukset
+TPM Missing / Disabled=TPM On Pois Päältä / Sitä Ei Löydy
+Detected=Löydetty
+DirectX 12 and WDDM 3=DirectX 12 and WDDM 3
+DirectX 12 and WDDM 2=DirectX 12 and WDDM 2
```
Comment (PriitUring) [inline Langs/040B.lang:45]: Should replace "and" with word "ja".

**Comment by micwoj92** [comment]
open pr

W dniu środa, 30 czerwca 2021 PriitUring ***@***.***>
napisał(a):

**Pull request merged**